src/cantools/database/can/formats/arxml/system_loader.py

```diff
@@ -1530,42 +1574,57 @@ def _load_signal(self, i_signal_to_i_pdu_mapping):
         )
         return signal
 
-    def _load_signal_name(self, i_signal):
+    def _load_signal_name(self, i_signal: ElementTree.Element) -> str:
         system_signal_name_elem = \
             self._get_unique_arxml_child(i_signal,
                                          [
                                              '&SYSTEM-SIGNAL',
                                              'SHORT-NAME'
                                          ])
-        if system_signal_name_elem is not None and len(system_signal_name_elem):
+        if system_signal_name_elem and system_signal_name_elem.text:
             return system_signal_name_elem.text
 
-        return self._get_unique_arxml_child(i_signal, 'SHORT-NAME').text
-
-    def _load_signal_start_position(self, i_signal_to_i_pdu_mapping):
-        pos = self._get_unique_arxml_child(i_signal_to_i_pdu_mapping,
-                                           'START-POSITION').text
-        return parse_number_string(pos)
-
-    def _load_signal_length(self, i_signal, system_signal):
-        i_signal_length = self._get_unique_arxml_child(i_signal, 'LENGTH')
+        signal_name_elem = self._get_unique_arxml_child(i_signal, 'SHORT-NAME')
+        if signal_name_elem and signal_name_elem.text:
+            return signal_name_elem.text
+        else:
+            # TODO Determine if signals should be able to be missing a name
```

Comment:
IIRC, the AUTOSAR XSD does not allow this. that said, IMO we should try to accommodate non-conforming datasets on a best-effort basis...